web_sdk/BUILD.gn

```diff
@@ -551,3 +564,59 @@ _kernel_worker("flutter_dartdevc_kernel_sdk_outline_sound") {
     "dart:_engine",
   ]
 }
+
+# Archives Flutter Web SDK
+zip_bundle("flutter_web_sdk_archive") {
+  output = "flutter-web-sdk-${host_os}-${target_cpu}.zip"
```

Comment:
The `${host_os}-${target_cpu}` combo doesn't make sense. I think you want `${target_os}-${target_cpu}`. Not sure why the `${target_cpu}` part is included since I think these artifacts are arch generic, but maybe @yjbanov can explain.

If this is going to be coming up more even after this, we could consider making some of the definitions here public: https://github.com/flutter/engine/blob/main/common/config.gni#L87

Reply:
There are many artifacts for the different platforms that are using `${target_os}-${target_cpu}` in their names. I'll go ahead and make the definition you pointed out public.